tower-http/Cargo.toml

```
@@ -16,7 +16,7 @@ bytes = "1"
 futures-core = "0.3"
 futures-util = { version = "0.3", default_features = false, features = [] }
 http = "0.2"
-http-body = "0.4.1"
+http-body = "0.4"
```

This needs to remain 0.4.1 since tower-http uses features shipped in 0.4.1. If we only depend on 0.4 a user might get 0.4.0 which would lead to compile errors.

```suggestion
http-body = "0.4.1"
```